ipwb/indexer.py

```diff
@@ -170,6 +170,83 @@ def indexFileAt(warcPaths, encryptionKey=None,
         print('\n'.join(cdxjLines))
 
 
+def generateCDXJMetadata(cdxjLines=None):
+    metadata = ['!context ["http://tools.ietf.org/html/rfc7089"]']
```

Comment:
I remember mentioning somewhere that we need to change this context from the RFC to a separate URL (was that comment on your JCDL submission draft?).

Reply:
Perhaps but per below, this code is not new to this PR.